File: C/PRAK603-22101817310001-MuhammadRyanRizkyRahmadi.c
#include <stdio.h>
int main(){
    int r, y, angka, a;
    scanf("%d %d", &r, &y);
    if(r!=y){
        printf("Jumlah tidak sama");
    }else{
    int barisr[r];int barisy[y];
    for(a = 0; a < r; a++){
        scanf("%d", &barisr[a]);} 
    for (a = 0; a < y; a++){
        scanf("%d", &barisy[a]);} 
    for(a = 0; a < r; a++){
        angka = barisr[a] * barisy[a];
        printf("%d ", angka);}
    }
}

File: Python/PRAK603-22101817310001-MuhammadRyanRizkyRahmadi.py
r, y = map(int, input().split())
if (r != y):
    print('Jumlah tidak sama')
else:
    barisr = list(map(int, input().split()))
    barisy = list(map(int, input().split()))
    for i in range (r):
        print(barisr[i] * barisy[i], end=' ')